ember-flight-icons/jsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
[personal opinion alert] personal editor config should generally not be committed to a shared repo

Reply:
The only reason why I committed this file is because it didn’t work without it and the Ember console message said to add it. This made me wonder if we would be able to deploy a working page if this wasn’t in the repo. What do you think?

Reply:
That sounds 🐟 -y since this is only for development with vs code. I'd be happy to take a look to figure out what might be causing the errors locally.

Reply:
let me make sure it's not just me, first! 👍

Reply:
it's unclear to me what the decision is here. it sounds like the decision to remove this file before merge? 

(blocking) please make that change before +1